SeleniumBasicsTest/appSource/app_tecklist.py

```diff
@@ -0,0 +1,72 @@
+from selenium.webdriver.support.select import Select
+
+from libCommon.selenium_helper import SeleniumHelper
+import time
+from sourcelocators.techlocators import Techlistlocators
+
+class APPTechList:
+
+    def __init__(self):
+        self.url = "https://www.techlistic.com/p/selenium-practice-form.html"
+        self.driver = SeleniumHelper()
+        self.driver.implecit_waits(10)
+
+    def launch(self):
+
+        self.driver.launch_app(self.url)
+
+        self.driver.max_window()
+
+        self.driver.execute_script("1100")
+
+        time.sleep(5)
+        try:
+            firstname = self.driver.find_element(Techlistlocators.first_name)

```

Comment:
please give proper function name ex: get_element instead find_element

Reply:
i will change as per the error and i will PR
thanks very much